teku/src/test/java/tech/pegasys/teku/cli/options/ValidatorOptionsTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.cli.options;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import org.apache.tuweni.bytes.Bytes32;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.teku.bls.BLSPublicKey;
+import tech.pegasys.teku.cli.AbstractBeaconNodeCommandTest;
+import tech.pegasys.teku.datastructures.util.DataStructureUtil;
+import tech.pegasys.teku.util.config.TekuConfiguration;
+
+public class ValidatorOptionsTest extends AbstractBeaconNodeCommandTest {
+  private final DataStructureUtil dataStructureUtil = new DataStructureUtil();
+  private final Bytes32 graffiti =
+      Bytes32.fromHexString("0x542045204b205500000000000000000000000000000000000000000000000000");
```

Comment:
is this the hex of the "T E K U" string that is in the config file? Can the test verify this?

Reply:
its annoying, without converting the string with the same functionality in the converter, which would just be duplicating the code. I manually verified with a Hex->utf8 page.
The same logic applies for the test cases for the converter itself, its not a trivial function to convert the text to hex (at least that i found)

Reply:
a comment in the test would make it clearer then

Reply:
moved it to `shouldReadFromConfigurationFile` and added comment